teamworkApp/lib/studentGraph.py

```diff
@@ -1,53 +1,54 @@
 import argparse 
 import matplotlib.pyplot as plt
 import numpy as np
+import sys
 
 import dbCalls
 from evaluateAnswers import Style
 
-def student_graph(student_id):
+
+def student_graph(student_ids):
     """Returns a path to where the student's graph is saved"""
 
     labels = [Style(x).name for x in range(len(Style))]
-
     # Connect to db to get scores
-    dbData = dbCalls.get_students_styles([student_id])
-    
+    dbData = dbCalls.get_students_styles(student_ids)
+    yPos = np.arange(len(labels))
+    scoreSet = [data[2:6] for data in dbData]
+
     # Get the scores needed
     assert(len(dbData) != 0), "Cannot find student data."
-    scores = dbData[0][2:6]
 
-    # Make graph
-    labels = [Style(x).name for x in range(len(Style))]
-    yPos = np.arange(len(labels))
-    plt.bar(yPos, scores, align = 'center', alpha = 0.5)
-    plt.xticks(yPos, labels)
-    plt.ylabel("Score")
-    plt.title("Student " + str(student_id) + " Distribution of Types")
+    # create split bar graph if <= 4 students
+    if len(scoreSet) <= 6:
+        # change for each set of bars
+        widthChange = -0.4
+        width = 0.8/len(scoreSet)
+        colors = ['#3fe0d0', '#111e6c', '#008081', '#a3de38', '#3378e8', '#8733e8']
+        currIter = 0
+        for scores in scoreSet:
+            username = dbCalls.get_name(student_ids[currIter])
```

Comment:
It would be more efficient to write a dbCall that fetches the names of all of the students rather than just one.

Reply:
@reagan-smith can you fix this within my branch? I'm not familiar enough with your code